site/content/how-to/control-plane-configuration.md

```diff
@@ -22,42 +22,22 @@ If the resource is invalid to the OpenAPI schema, the Kubernetes API server will
 
 Additionally, the control plane updates the status of the resource (if it exists) to reflect whether it is valid or not.
 
-### Spec
-
-{{< bootstrap-table "table table-striped table-bordered" >}}
-| name    | description                                                     | type                     | required |
-|---------|-----------------------------------------------------------------|--------------------------|----------|
-| logging | Logging defines logging related settings for the control plane. | [logging](#speclogging) | no       |
-{{< /bootstrap-table >}}
-
-### Spec.Logging
-
-{{< bootstrap-table "table table-striped table-bordered" >}}
-| name  | description                                                            | type   | required |
-|-------|------------------------------------------------------------------------|--------|----------|
-| level | Level defines the logging level. Supported values: info, debug, error. | string | no       |
-{{< /bootstrap-table >}}
+**For a full list of configuration options that can be set, see the `NginxGateway spec` in the [API reference]({{< relref "reference/api.md" >}}).**
 
 ## Viewing and Updating the Configuration
 
-{{< note >}} For the following examples, the name `nginx-gateway-config` should be updated to the name of the resource created for your installation. {{< /note >}}
+{{< note >}} For the following examples, the name `ngf-config` should be updated to the name of the resource created for your installation.{{< /note >}}
```

Comment:
Is this already done for the user? / Is this the default name?

Otherwise, I think it would be helpful to show the command to update the name.

Reply:
It just means when they run kubectl commands, change the name of the resource that they are editing if they created their resource with a different name.

Reply:
Oh actually I misread, I thought this line meant to update the resource to this name for some reason.